about.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Inti Raymi Music Festival</title>
+    <link rel='stylesheet' href='https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css'>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="about.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+
+  <section class="top-page">
+    <div class="session-container">
+      <ul id="session-social" class="d-flex mx-4 mt-2 mb-1">
+       <li>
+        <a class="sessionbar px-4" href ="#">
+          <i>
+            <img class="sessionbar w-25" src="./capstoneimgs/facebook.png" alt="facebook-link">
+              </i>
+         </a>
+      </li>
+      <li>
+        <a class="sessionbar px-4" href="#">
+          <i>
+            <img class="sessionbar w-25" src="./capstoneimgs/twitter.png" alt="twitter-link">
+          </i>
+        </a>
+      </li>
+      <li class="sessionbar px-4">English</li>
+      <li class="sessionbar px-4">My Page</li>
+      <li class="sessionbar px-4">Logout</li>
+    </ul>
+  </div>
+
+  <a id="logonav" href="#top-page" class="sm d-none">
+    <img src="./capstoneimgs/logo.png" alt="world-music-festival-logo"></a>
+  <div class=" secondary-nav">
+   
+    <ul class="d-flex justify-content-end px-4 mt-3">
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="index.html">Home</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Program</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Performers</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Partners</a>
+      </li>
+      <li class="sec-nav-item-btn">
+        <button class="sec-nav-link-btn px-4">Campaign</button>
+      </li>
+    </ul>
+  </div>
+
+        <div id="header">
```

Comment:
Kindly use semantic header element instead of div with header id. `<header> </header>`

Reply:
Thank it's done